.travis.yml

```diff
@@ -1,36 +1,50 @@
 language: php
-
 php:
   - 7.1
   - 7.2
+  - nightly
+sudo: false
 
-matrix:
-  include:
-    - php: 7.1
-      env: ECS=true
-    - php: 7.1
-      env: PHPSTAN=true
+env:
 
-cache:
-  directories:
-    - $HOME/.composer/cache
+matrix:
+  fast_finish: true
+  allow_failures:
+    - php: nightly
 
 install:
-  - composer install --no-interaction
-  - if [[ $ECS != "" ]]; then composer create-project symplify/easy-coding-standard temp/ecs; fi
+  - composer install --no-interaction --prefer-dist --optimize-autoloader
 
-script:
-  - vendor/bin/phpunit --coverage-clover=coverage.xml -v
-  - if [[ $ECS != "" ]]; then temp/ecs/bin/ecs check src tests; fi
-  - if [[ $PHPSTAN != "" ]]; then vendor/bin/phpstan analyse src --level max --configuration phpstan.neon; fi
+jobs:
+  include:
+    - stage: test
+      script:
+      - vendor/bin/phpunit --no-coverage
+
+    - stage: coverage
+      php: 7.1
+      script:
+        - vendor/bin/phpunit
+      after_script:
+        - wget https://scrutinizer-ci.com/ocular.phar && php ocular.phar code-coverage:upload --format=php-clover build/logs/clover.xml
```

Comment:
Is there any Scrutinizer feature that is not covered by PHPStan?

It is practicall and much advanced open-source replacement for it.